civihr_employee_portal/templates/civihr-employee-portal-civihr-report-custom.tpl.php

```diff
@@ -6,25 +6,27 @@
 </div>
 <?php endif; ?>
 
-<ul class="nav nav-tabs nav-justified report-tabs">
+<ul class="nav nav-tabs nav-justified report-tabs panel chr_tabs__header">
```

Comment:
Just realized now that these are already Bootstrap's tabs. Why the need for a custom component (`.chr_tabs__header`)? Are the default tabs already in use anywhere in the ssp? If no, couldn't you customize those by overriding the default style? If yes, couldn't you add a modifier (say, `.nav-tabs-header`, and without adding any new classes for children element)?